reana_server/cli.py

```diff
@@ -20,7 +20,52 @@
 # granted to it by virtue of its status as an Intergovernmental Organization or
 # submit itself to any jurisdiction.
 
-"""REANA Workflow Controller command line interface."""
+"""
+The REANA Server offers a CLI for user account operations.
+
+Specifically, from the cli an administrtor
```

Comment:
Capital CLI, and administrtr -> administrator

Reply:
thanks